src/main/java/com/eviive/personalapi/controller/UserController.java

```diff
@@ -57,8 +67,16 @@ public ResponseEntity<CurrentUserDTO> current(@CurrentSecurityContext final Secu
         summary = "Login",
         responses = {
             @ApiResponse(responseCode = "200", description = "OK"),
-            @ApiResponse(responseCode = "400", description = "Bad Request"),
-            @ApiResponse(responseCode = "401", description = "Unauthorized")
+            @ApiResponse(
+                responseCode = "400",
+                description = "Bad Request",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            ),
+            @ApiResponse(
+                responseCode = "401",
+                description = "Unauthorized",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            )
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'annotation array initialization' child has incorrect indentation level 12, expected level should be 16.